hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/HFileOutputFormat2.java

```diff
@@ -324,14 +332,15 @@ public void write(ImmutableBytesWritable row, V cell) throws IOException {
               }
             }
           }
-          wl = getNewWriter(tableNameBytes, family, conf, favoredNodes);
+          wi = getNewWriter(tableNameBytes, family, conf, favoredNodes);
 
         }
 
         // we now have the proper WAL writer. full steam ahead
         PrivateCellUtil.updateLatestStamp(kv, this.now);
-        wl.writer.append((ExtendedCell) kv);
-        wl.written += length;
+        wi.writer.append((ExtendedCell) kv);
+        wi.written += length;
+        wi.maxSequenceId = Math.max(kv.getSequenceId(), wi.maxSequenceId);
```

Comment:
Any concerns that `Cell#getSequenceId` is removed in HBase 3? Any plans for how we should handle that?